samples/eglise-st-jean.json

```diff
@@ -0,0 +1,8 @@
+{
+    "_id":"ejrbzoefkznfkpz",
+    "dc_title":"Eglise St-Jean-au-Marché, Troyes",
+    "dc_creator": "Louis Magnie",
+    "dc_issued": "2016-07-02",
+    "text": "![](https://steatite.utt.fr/optimized/7b0006e98ec564021778cf585cae3adec07f4389)",
+    "dc_license": "https://creativecommons.org/licenses/by-sa/4.0/"
```

Comment:
The license is unknown. You should let it undefined (which means "All rights reserved").